lib/Comment/EditCommentForm.js

```diff
@@ -0,0 +1,145 @@
+import React, { useState, useRef } from 'react';
+import { arrayOf, bool, func, shape, string } from 'prop-types';
+import { Mention, MentionsInput } from 'react-mentions';
+import Avatar from '../Avatar';
+import AvatarInformation from '../Avatar/AvatarInformation';
+import ShortcutTrigger from '../ShortcutTrigger';
+import Button from '../Button';
+import { MetaDot } from './CommentMeta';
+
+export function EditCommentForm({
```

Comment:
Comments originally had a single component for creating a new comment and editing an existing one. I have no separated them out to different ones, so we can explicitly render the correct looking form.